homeassistant/components/onewire/sensor.py

```diff
@@ -108,6 +109,32 @@ def _get_sensor_precision_family_28(device_id: str, options: Mapping[str, Any])
             state_class=SensorStateClass.MEASUREMENT,
         ),
     ),
+    "20": tuple(
+        [
+            OneWireSensorEntityDescription(
+                key=f"latestvolt.{device_key}",
+                device_class=SensorDeviceClass.VOLTAGE,
+                entity_registry_enabled_default=False,
+                native_unit_of_measurement=UnitOfElectricPotential.VOLT,
+                read_mode=READ_MODE_FLOAT,
+                state_class=SensorStateClass.MEASUREMENT,
+                name=f"Latest Voltage {device_key}",
```

Comment:
We prefer to use sentence casing on names
```suggestion
                name=f"Latest voltage {device_key}",
```